modules/las/src/lib/parse-las.js

```diff
@@ -16,6 +16,7 @@ export default function parseLAS(arraybuffer, options = {}) {
   const {onProgress} = options;
   const {skip, colorDepth} = options.las || {};
 
+  // eslint-disable-next-line
```

Comment:
This is totally fine, thouth we normally take care to only disable the one rule: 
`// eslint-disable-next-line complexity` 

(That said, the message is of course an indicator that it could be a good idea to split up the function, and it seems that there is e.g. an opportunity to create a `detectTwoByteColors` helper. But I am fine with the code as is.)

Reply:
Sounds good to me, I rebased and pushed a version with the `detectTwoByteColors` helper.